src/state.rs

```diff
@@ -100,9 +96,9 @@ impl State {
     pub fn register_process(&mut self) {
         let process_name = "SeaOfStars.exe";
         // Find the Process
-        match Process::with_name(process_name, &mut self.process_list) {
-            Ok(process) => {
-                if self.context.process.is_none() {
+        if self.context.process.is_none() {
```

Comment:
These changes are unrelated to the fps clock, right?

Reply:
yes - this just reduces the amount of un-needed checks